spec/tests/requires.fmf

```diff
@@ -0,0 +1,29 @@
+summary: Packages required for the test execution
```

Comment:
One more thought/question here: So far we have all L1 attributes named in singular. Shall this be rather `require` to be consistent with other keys or shall we keep it consistent with `rpm` and `dnf` terminology, meaning "what the test requires"?

Reply:
I think it's not plural, but rather: `Third-person singular simple present indicative form of require` ([src](https://www.wordhippo.com/what-is/the-meaning-of-the-word/requires.html)), which is fine with me. :)

Reply:
Yeah, I see. It can be both. Just experimented how `require` would look like in the config and it looks good as well. And it shorter ;-)

Reply:
Going for now with the shorter version, let's see how it works.